tools/oas-apigee-mock/lib/util/service.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+exports.servicesToArray = function (api) {
+  var services = []
+  if (Array.isArray(api['x-a127-services'])) {
```

Comment:
Are these `x-a127` service annotations documented somewhere? Maybe we can use this to clean up the old a127 name as well?